llm_toolkit/prompt_builder.py

```diff
@@ -211,26 +208,20 @@ def _add_examples(self,
       self._prompt.add_problem(problem)
       self._prompt.add_solution(solution)
 
-  def build(
-      self,
-      function_signature: str,
-      target_file_type: FileType,
-      example_pair: list[list[str]],
-      project_example_content: Optional[list[list[str]]] = None,
-      project_context_content: Optional[Tuple[str,
-                                              str]] = None) -> prompts.Prompt:
+  def build(self,
+            function_signature: str,
+            target_file_type: FileType,
+            example_pair: list[list[str]],
+            project_example_content: Optional[list[list[str]]] = None,
+            project_context_content: Optional[str] = None) -> prompts.Prompt:
```

Comment:
I presume `Optional` is not required for `project_context_content`, because it will be `''` if `use_context` is false? E.g.,
```python
project_context_content: str = ''
```

Reply:
that sounds right. fixing